src/gui/symbols/point_symbol_editor_widget.cpp

```diff
@@ -78,12 +80,13 @@
 
 namespace OpenOrienteering {
 
-PointSymbolEditorWidget::PointSymbolEditorWidget(MapEditorController* controller, PointSymbol* symbol, SymbolRole role, qreal offset_y, QWidget* parent)
+PointSymbolEditorWidget::PointSymbolEditorWidget(MapEditorController* controller, PointSymbol* symbol, Map* source_map, SymbolRole role, qreal offset_y, QWidget* parent)
```

Comment:
Early feedback: 
I don't think we should add another parameter. The source map is implicitly known, via controller and symbol.
(Another suggested workflow was copy-and-paste. This would use a different Map, with all complications. And you wouldn't get that Map at constuction time, before "Paste".)

Reply:
I also didn't like to add a new parameter but had not found a better alternative.
The newly created controller being passed to PointSymbolEditorWidget only knows the preview map but does not know the source_map, neither does does the symbol. There is certainly the previously active controller who 'knows' the source map but I don't know how PointSymbolEditorWidget can access it.
MapEditorController describes itself as a 'HACK' for being used in the symbol editor for the preview. MapEditorController could be extended to store the source map in addition but that only extends the hack.